### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/response-cookies.md`

### PR DESCRIPTION
**Pull request by Paulofalcao2002**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]
Good work, @Paulofalcao2002. The community thank you for dedication, we hope to work with for longer.

**Review by ceb10n — APPROVED** [review]
Great work @Paulofalcao2002  🚀

File: docs/pt/docs/advanced/response-cookies.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Portanto, você terá que garantir que seus dados sejam do tipo correto. E.g. será compatível com JSON se você estiver retornando um `JSONResponse`.
+
+E também que você não esteja enviando nenhum dado que deveria ter sido filtrado por um `response_model`.
+
+///
+
+### Mais informações
+
+/// note | "Detalhes Técnicos"
```
Comment (andersonrocha0) [inline docs/pt/docs/advanced/response-cookies.md:45]: I would suggest to use "notas" instead of "detalhes técnicos"?

**Review by andersonrocha0 — CHANGES_REQUESTED** [review]

File: docs/pt/docs/advanced/response-cookies.md (hunk 1 above)
Comment (Paulofalcao2002) [reply docs/pt/docs/advanced/response-cookies.md:45]: The title of this note in `docs/en/docs/advanced/response-cookies.md` is:

`/// note | "Technical Details"`

So, I translated it to:

`/// note | "Detalhes Técnicos"`

Does it make sense or should I change it to "notas" ?

**Review by andersonrocha0 — APPROVED** [review]

File: docs/pt/docs/advanced/response-cookies.md (hunk 1 above)
Comment (andersonrocha0) [reply docs/pt/docs/advanced/response-cookies.md:45]: It's good. We have more than one way.

**Comment by alejsdev** [comment]
Thank you! @andersonrocha0 :tada: :sparkles:

**Pull request merged**